@here/harp-materials/lib/SolidLineMaterial.ts

```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { LineCaps } from "@here/harp-datasource-protocol";
```

Comment:
I really don't like that harp-materials depends on harp-datasource-protocoll just b/c of the caps and dash enums. I would rather redefine the enums here and rely on the tests that they match

Reply:
Will do

Reply:
Done

Reply:
Since the latest (succesfully reviewed) changes still have a dependency on datasource-protocol, I reverted to use the existing scheme and keep that dependency.